Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:7
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+COPY package.json .
+COPY . .
+RUN yarn install
```

Comment:
To avoid docker cache invalidation, do `yarn install` before the second `COPY` command.